multiregion/README.md

```diff
@@ -282,20 +292,27 @@ Topic: multi-region-sync	PartitionCount: 1	ReplicationFactor: 4	Configs: min.ins
 
 Topic: multi-region-async	PartitionCount: 1	ReplicationFactor: 4	Configs: min.insync.replicas=1,confluent.placement.constraints={"version":1,"replicas":[{"count":2,"constraints":{"rack":"west"}}],"observers":[{"count":2,"constraints":{"rack":"east"}}]}
 	Topic: multi-region-async	Partition: 0	Leader: none	Replicas: 2,1,3,4	Isr: 1	Offline: 2,1	Observers: 3,4
+
+==> Describe topic multi-region-default
+
+Topic: multi-region-default	PartitionCount: 1	ReplicationFactor: 4	Configs: min.insync.replicas=1,confluent.placement.constraints={"version":1,"replicas":[{"count":2,"constraints":{"rack":"west"}}],"observers":[{"count":2,"constraints":{"rack":"east"}}]}
+	Topic: multi-region-default	Partition: 0	Leader: none	Replicas: 2,1,3,4	Isr: 1	Offline: 2,1	Observers: 3,4
 ```
 
 Observations:
 
 * In the first case, the topic `single-region` has no leader, because it had only two replicas in the ISR, both of which were in the `west` region and are now down.
 * In the second case, the topic `multi-region-sync` automatically elected a new leader in `east` (e.g. replica 3 in the above output).  Clients can failover to those replicas in the east region.
-* In the third case, the topic `multi-region-async` also has no leader, because it had only two replicas in the ISR, both of which were both in the `west` region and are now down.  The observers in the `east` region are not eligible to become leaders automatically because they were not in the ISR.
+* In the third case, the topics `multi-region-async` and `multi-region-default` also have no leader, because they had only two replicas in the ISR, both of which were in the `west` region and are now down.  The observers in the `east` region are not eligible to become leaders automatically because they were not in the ISR.
```

Comment:
```suggestion
* In the last two cases, the topics `multi-region-async` and `multi-region-default` also have no leader, because they had only two replicas in the ISR, both of which were in the `west` region and are now down.  The observers in the `east` region are not eligible to become leaders automatically because they were not in the ISR.
```